### ISSUE TITLE
Return cached post-dated L1 closing transaction on `newBid` to Frontend CLI

### ISSUE THREAD

**Issue by uhbif19** [mlabs-haskell/hydra-auction#159]
Full requirements are in spec

Blocked by #158

**Comment by uhbif19** [comment]
I missed that to do that we need use private Hydra API to construct tx. That is out of scope of our project. Will ask Hydra for that feature and block issue until that.

**Comment by uhbif19** [comment]
Hydra team said that implement time on L2 would be simpler and they already planned that. So we should use time API, but this will be probably implemented only after delivery of current project.
